.travis.yml

```diff
@@ -140,7 +140,7 @@ install:
       pushd "$HOME"
       git clone --single-branch --branch RC_2_0 https://github.com/arvidn/libtorrent.git
       cd libtorrent
-      git checkout tags/v2.0.2
+      git checkout tags/v2.0.3
```

Comment:
Accidental unwanted change?

Reply:
Why unwanted? file_prio_alert is existing since libtorrent-2.0.3.

Reply:
But the PR will work even without this bump. Shouldn't you only bump it when the minimum requirement is updated?

Reply:
Really there are some other improvements related to "resume data" handling in libtorrent 1.2.13/2.0.3 (also contributed by me) so I would still require qBittorrent-4.4 to raise libtorrent version requirements.
Of course I can do it all at once in separate PR.